build/verifyids.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/python
+
+import sys
+
+# Parse /etc/passwd file and provide
+# put data into array
+passwd = open( "/etc/passwd", "r" )
+idstable = open( "/root/ids.tables", "r" )
+array = []
+ids_table = []
+for line in passwd:
+    tokens = line.split(":")
+    user_uid_gid = {'user': tokens[0], 'uid': tokens[2], 'gid': tokens[3]}
+    array.append(user_uid_gid)
+for line in idstable:
+    ids_table.append(line.strip())
+ids_table = "".join(ids_table)
+passwd.close()
+idstable.close()
+
+user_list = sorted(array, key=lambda k: k['user'])
+
+# Write uids list
+etc_passwd =  "uids = {'" + user_list[0]['user'] + "': ('" + user_list[0]['uid'] + "', '" + user_list[0]['gid'] + "'),"
+for user in user_list[1:-1]:
+  etc_passwd += " '" + user['user'] + "': ('" + user['uid'] + "', '" + user['gid'] + "'),"
+etc_passwd += " '" + user_list[-1]['user'] + "': ('" + user_list[-1]['uid'] + "', '" + user_list[-1]['gid'] + "')}"
+
+# Write gids list
+etc_passwd += "gids = {'" + user_list[0]['user'] + "': ('" + user_list[0]['gid'] + "', ''),"
+for user in user_list[1:-1]:
+  etc_passwd += " '" + user['user'] + "': ('" + user['gid'] + "', ''),"
+etc_passwd += " '" + user_list[-1]['user'] + "': ('" + user_list[-1]['gid'] + "', '')}"
+
+if ids_table == etc_passwd:
+  sys.exit(0)
+sys.exit(1)
```

Comment:
I would add debug here to have details of what is failing to easily add the UID/GID in ids.tables.